docs/traffic-policy/actions/add-headers/index.mdx

```diff
@@ -0,0 +1,5 @@
+## Overview
+
+The **Add Headers** Traffic Policy action enables you to add headers to an HTTP 
+request before it is sent upstream or an HTTP response before it is sent back to 
```

Comment:
nit: "or an HTTP response" -> "or to an HTTP response"

Reply:
not needed since it's proceeded by a to already, I originally had this and Grammarly smacked my hand